Sources/Commands/PackageTools/Update.swift

```diff
@@ -41,16 +42,18 @@ extension SwiftPackageTool {
                 observabilityScope: swiftTool.observabilityScope
             )
             
-            // try to load the graph which will emit any errors
-            if !swiftTool.observabilityScope.errorsReported {
-                _ = try workspace.loadPackageGraph(
-                    rootInput: swiftTool.getWorkspaceRoot(),
-                    observabilityScope: swiftTool.observabilityScope
-                )
+            do {
```

Comment:
Does this `do` block have any effect without `catch`?

Reply:
its to scope the `start` variable

Reply:
Sorry, I don't see `start` declared in this block, but I see it declared in other functions. Could that be missing here by any chance, or am I misunderstanding how `do` can be applied?

Reply:
oops you are right, I removed that code, so that is left over